.github/workflows/ci.yml

```diff
@@ -31,16 +31,40 @@ jobs:
 
       - name: Build
         run: npm run build
+
+      - name: Pack
+        run: npm pack
+
+      - name: Upload package artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: package
+          path: cldn-components-0.0.0-dev.tgz
```

Comment:
`0.0.0-dev` is fixed in `package.json` and not modified. As seen in the workflow, `npm version ...` sets the version before publishing. The changed version is not committed to the repository because it's not needed.